### DISCUSSION TITLE
Why does it use caddy by default ?

### DISCUSSION THREAD

**Discussion by Greatz08** [bluesky-social/atproto#2898] [Clarifications and "Why?"]
> Note: there is no need to set up TLS or redirect requests from port 80 to 443 because the Caddy web server, included in the Docker compose file, will handle this for you.

This is mentioned in self-hosting section so i wanna know the reason for choosing caddy over nginx or traefik ? And can you create seperate docker compose file which people can use without having to depend on caddy ?

**Comment by bnewbold** [comment]
Currently we want to keep the instructions and maintenance very very simple, so we are supporting only single very rigid configuration.

I think the reason we use Caddy is a mix of simple auto-TLS support (with let's encrypt), and the original developer just liking Caddy at the time.
